src/FolderConfig.cpp

```diff
@@ -0,0 +1,92 @@
+#include "FolderConfig.h"
+#include <QFile>
+#include <QDebug>
+#include<QDir>
+
+namespace QtDBMigrationNS {
+FolderConfig::FolderConfig(const QString &path): AbstractConfig(path)
+{
+}
+
+QString getOperation(QString line, QString operation)
+{
+    QRegularExpression rxOperation("^-- \\+migrate (?<operation>(Up|Down))");
+    QRegularExpressionMatch operationMatch = rxOperation.match(line);
+    if (operationMatch.hasMatch())
+    {
+        operation = operationMatch.captured("operation");
+    }
+    return operation;
+}
+
+
+bool FolderConfig::load()
+{
+    QRegExp rx("^\\d{4}\\.sql$");
+    QDir dir(m_path);
+    QStringList nameFilter("*.sql");
+    QStringList sqlFiles = dir.entryList(nameFilter);
+
+    int totalFiles = sqlFiles.count();
+
+    if (!dir.exists() || totalFiles < 1) {
+        return false;
+    }
+
+    for(auto fileName : sqlFiles) {
+        QString operation = "";
+        QString filePath = dir.filePath(fileName);
+        QFile file(filePath);
+        if(!file.exists())
+        {
+            return false;
+        }
+        QFileInfo fileInfo(file.fileName());
```

Comment:
Why do you need `QFileInfo`?

Reply:
Used it more at first, now I see it is only in one statement and not really needed. Removed it.